mypyc/genops.py

```diff
@@ -532,9 +567,39 @@ def visit_name_expr(self, expr: NameExpr) -> int:
             self.add(Assign(target, reg))
             return target
 
+    def visit_member_expr(self, expr: MemberExpr) -> int:
+        target = self.alloc_target(type_to_rttype(self.types[expr]))
+        module = '.'.join(expr.node.fullname().split('.')[:-1])
+        right = expr.node.fullname().split('.')[-1]
+        left = self.alloc_temp(RTType('object'))
+        self.add(ModuleGet(left, module))
+        self.add(GetAttr(target, left, right))
+        return target
+        
     def visit_call_expr(self, expr: CallExpr) -> int:
+        # Unfortunately both
+        #   some_list.append(..)
+        # and
+        #   some_module.some_function(..)
+        #
+        # Look precisely the same here. The second condition is kind of garbage
+        # but seems to work.
```

Comment:
Here's what may work:
* First try to generate a special method call. If this succeeds, we are done.
* If special method call generation returned an error code, box the callee (as needed) and arguments, and generate a `PyCall`.